alot/account.py

```diff
@@ -348,7 +348,9 @@ def __init__(self, cmd, **kwargs):
 
         try:
             # make sure self.mail is a string
-            out, *_ = await call_cmd_async(cmdlist, stdin=str(mail))
+            out, err, code = await call_cmd_async(cmdlist, stdin=str(mail))
+            assert code == 0, \
+                "command exited with code {}.\n{}".format(code, err)
```

Comment:
I don't think an assert is the right way to handle this. Asserts are for cases that must be true, and not being true is an unrecoverable error, they also don't get run if you use -O. So I think we should do an `if code != 0: raise` instead.